pcsx2/gui/AppMain.cpp

```diff
@@ -236,8 +291,9 @@ void Pcsx2App::PadKeyDispatch( const keyEvent& ev )
 	m_kevt.SetEventType( ( ev.evt == KEYPRESS ) ? wxEVT_KEY_DOWN : wxEVT_KEY_UP );
 	const bool isDown = (ev.evt == KEYPRESS);
 
+//returns 0 for normal keys and a WXK_* value for special keys
 #ifdef __WXMSW__
-	const int vkey = wxCharCodeMSWToWX( ev.key );	//returns 0 if plain ascii value or a WXK_... (<=32 or >=300) if a special key
+	const int vkey = TranslateVKToWXK(ev.key);
```

Comment:
Why did you add your own translation function? It seems to me that according to http://trac.wxwidgets.org/ticket/12508 , the only difference between the new `VKToWX` to the old `wxCharCodeMSWToWX` is that the new one doesn't return 0, i.e. it returns either an ascii code or the special code, at which case I'd _think_ it should work as is by calling `VKToWX` on wx3.x, since the rest of the code does `m_kevt.m_keyCode = vkey? vkey : ev.key;` which should work either for the old wx call or the new one.

Am I wrong?